src/axom/bump/CMakeLists.txt

```diff
@@ -0,0 +1,124 @@
+# Copyright (c) 2017-2025, Lawrence Livermore National Security, LLC and
+# other Axom Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+#------------------------------------------------------------------------------
+# BUMP -- Blueprint Utilities for Mesh Processing
+#------------------------------------------------------------------------------
+
+#------------------------------------------------------------------------------
+# Check necessary dependencies
+#------------------------------------------------------------------------------
+axom_component_requires(NAME       BUMP
+                        COMPONENTS SLIC SLAM PRIMAL
```

Comment:
@BradWhitlock follow-up to my comment above....if this line indicates the Axom dependencies for bump, then shoudln't the `axom_add_component` call for bump come before the `axom_add_component` call for sidre?

Reply:
I moved bump/mir to after spin in that list of `axom_add_component` calls.

Reply:
Aaaand it looks like I need to add spin there.